### PR TITLE
Feat: multicluster vela status/exec/port-forward

### PR DESCRIPTION
**Pull request by Somefive**
### Description of your changes

Support exec for pods in remote cluster. Support port-forward for workloads/service in remote cluster. Continue from #2658.

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [x] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Review by wonderflow — APPROVED** [review]
LGTM, please fix CI

**Review by hongchaodeng — APPROVED** [review]

File: pkg/clustermanager/cluster_manager.go [hunk 1]
```diff
@@ -56,10 +56,14 @@ func GetRegisteredClusters(c client.Client) ([]types2.Cluster, error) {
 		return nil, errors.Wrapf(err, "failed to get clusterSecret secrets")
 	}
 	for _, clusterSecret := range secrets.Items {
+		endpoint := string(clusterSecret.Data["endpoint"])
+		if endp, ok := clusterSecret.GetLabels()[v1alpha1.LabelKeyClusterEndpointType]; ok {
+			endpoint = endp
```
Comment (zzxwill) [inline pkg/clustermanager/cluster_manager.go:61]: Why do we have two sources for `endpoint`?

Comment (Somefive) [reply pkg/clustermanager/cluster_manager.go:61]: ClusterGateway undergoes a version update which means we need to support backward-compatibility.

**Review by zzxwill — APPROVED** [review]
/lgtm

**Pull request merged**